cocos/navmesh/CCNavMesh.cpp

```diff
@@ -32,20 +32,22 @@
 
 NS_CC_BEGIN
 
+#pragma pack(push,1)
 struct TileCacheSetHeader
 {
-    int magic;
-    int version;
-    int numTiles;
+    int32_t magic;
+    int32_t version;
+    int32_t numTiles;
```

Comment:
And i think there are many places that read a file and parse it, such as tilemap xml file, cocos builder output file and so on. Should we check them too?